pennylane/qnode.py

```diff
@@ -28,7 +28,12 @@
 from pennylane.operation import State
 
 from pennylane.interfaces.autograd import AutogradInterface, np as anp
-from pennylane.tape import JacobianTape, QubitParamShiftTape, CVParamShiftTape, ReversibleTape
+from pennylane.tape import (
+    JacobianTape,
+    QubitParamShiftTape,
+    CVParamShiftTape,
+    ReversibleTape,
+)

```

Comment:
@paniash since this line hasn't been modified, it looks like `black` might have been run with a different line setting 🤔 

Could you re-run `black`, but with the line length set to 100? This is the standard we use in the repository to minimize formatting changes:

```python
black -l 100 pennylane
```

Reply:
Hi @paniash, I'm unresolving this one, since it seems this change is still here 🤔 

Did running `black` with a line length of 100 not fix it?

Reply:
@josh146 I ran `black -l 100 pennylane/qnode.py` and it shows "1 file left unchanged".

Reply:
Ah, that's very odd 🤔 
Could you try updating your version of black (`pip install black --upgrade`) and trying again?

Reply:
Oh, nevermind, the CI check below seems to be passing, so this should be fine 🙂